packages/libretro/neocd/package.mk

```diff
@@ -0,0 +1,45 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.tv; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+################################################################################
+
+PKG_NAME="neocd"
+PKG_VERSION="b90144e"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="LGPLv3"
+PKG_SITE="https://github.com/libretro/neocd_libretro"
+PKG_URL="$PKG_SITE.git"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_PRIORITY="optional"
+PKG_SECTION="libretro"
+PKG_SHORTDESC="Neo Geo CD emulator for libretro"
+PKG_LONGDESC="Neo Geo CD emulator for libretro"
+PKG_TOOLCHAIN="make"
+
+PKG_IS_ADDON="no"
+PKG_AUTORECONF="no"
+
+pre_configure_target() {
+  cd "$PKG_BUILD"
```

Comment:
is that necessary?

Reply:
Yes otherwise I got this error

```
DISTRO=Lakka PROJECT=Generic ARCH=x86_64 scripts/build neocd
BUILD      neocd (target)
    TOOLCHAIN      make
Executing (target): make 
make: *** No targets specified and no makefile found.  Stop.
```